src/Serilog/Formatting/Display/LogEventLevelValue.cs

```diff
@@ -0,0 +1,116 @@
+// Copyright 2013-2015 Serilog Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+using Serilog.Events;
+using Serilog.Parsing;
+
+namespace Serilog.Formatting.Display
+{
+    // allows for the specific handling of the {Level} element.
+    // can now have a fixed width applied to it, as well as casing rules.
+    internal class LogEventLevelValue : LogEventPropertyValue
+    {
+        private readonly LogEventLevel _value;
+
+        // Assumes reading from Dictionary<,> is thread-safe after construction
+        private static readonly Dictionary<LogEventLevel, string[]> _shortenedLevelMap
+            = new Dictionary<LogEventLevel, string[]>
+                  {
+                      { LogEventLevel.Verbose, new []{ "V", "Vb", "Vrb", "Verb" } },
+                      { LogEventLevel.Debug, new []{ "D", "De", "Dbg", "Dbug" } },
+                      { LogEventLevel.Information, new []{ "I", "In", "Inf", "Info" } },
+                      { LogEventLevel.Warning, new []{ "W", "Wn", "Wrn", "Warn" } },
+                      { LogEventLevel.Error, new []{ "E", "Er", "Err", "Eror" } },
+                      { LogEventLevel.Fatal, new []{ "F", "Fa", "Ftl", "Fatl" } },
+                  };
```

Comment:
One other option here that will be a bit more efficient:

```csharp
        static readonly string[][] _shortenedLevelMap
            = new string[][]
                  {
                      new []{ "V", "Vb", "Vrb", "Verb" },
                      new []{ "D", "De", "Dbg", "Dbug" },
                      new []{ "I", "In", "Inf", "Info" },
                      new []{ "W", "Wn", "Wrn", "Warn" },
                      new []{ "E", "Er", "Err", "Eror" },
                      new []{ "F", "Fa", "Ftl", "Fatl" },
                  };
```

And then:

```csharp
        static string ShortLevelFor(LogEventLevel value, int width)
        {
            var idx = (int)value;
            if (idx < 0 || idx > (int)(LogEventLevel.Fatal))
                return "";
            return _shortenedLevelMap[idx][width - 1];
        }
```

The check might be needed even in the dictionary-based code, since it's possible (although) unlikely, to get a level like `((LogEventLevel) 42)`.